ding/reward_model/gail_irl_model.py

```diff
@@ -166,10 +166,10 @@ def estimate(self, data: list) -> None:
         res = concat_state_action_pairs(data)
         res = torch.stack(res).to(self.device)
         with torch.no_grad():
-            reward = -self.reward_model(res).squeeze(-1).cpu()
+            reward = self.reward_model(res).squeeze(-1).cpu()
         reward = torch.chunk(reward, reward.shape[0], dim=0)
         for item, rew in zip(data, reward):
-            item['reward'] = rew
+            item['reward'] = -torch.log(rew)
```

Comment:
shall we add a small number in log to avoid potential explosion? e.g. -torch.log(rew + 1e-8)

Reply:
It makes sense, I will add it, thank you for your finding.